src/main/java/com/kainos/ea/util/DatabaseConnector.java

```diff
@@ -0,0 +1,50 @@
+package com.kainos.ea.util;
+
+import com.kainos.ea.exception.DatabaseConnectionException;
+
+import java.io.FileInputStream;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.util.Properties;
+
+
+public class DatabaseConnector {
+
+    private static Connection conn;
+
+    public Connection getConnection() throws DatabaseConnectionException {
+        String user;
+        String password;
+        String host;
+        String db;
+
+        if ( conn != null ) return conn;
+
+        try {
+
+            FileInputStream propsStream =
+                    new FileInputStream("./src/main/resources/db.properties");
+
+            Properties props = new Properties();
+            props.load(propsStream);
+
+            user            = props.getProperty("user");
```

Comment:
Spacing here looks off but it might just be how it's displayed. Ensure you've no additional whitespace after the variable name